src/openrct2-ui/windows/Guest.cpp

```diff
@@ -2222,3 +2335,97 @@ void window_guest_inventory_paint(rct_window* w, rct_drawpixelinfo* dpi)
         gfx_draw_string_left(dpi, STR_NOTHING, nullptr, COLOUR_BLACK, x, y);
     }
 }
+
+void window_guest_debug_resize(rct_window* w)
+{
+    window_set_resize(w, 192 + TabWidth, 159, 500, 450);
+}
+
+void window_guest_debug_invalidate(rct_window* w)
+{
+    if (window_guest_page_widgets[w->page] != w->widgets)
+    {
+        w->widgets = window_guest_page_widgets[w->page];
+        window_init_scroll_widgets(w);
+    }
+
+    w->pressed_widgets |= 1ULL << (w->page + WIDX_TAB_1);
+
+    rct_peep* peep = GET_PEEP(w->number);
+
+    set_format_arg(0, rct_string_id, peep->name_string_idx);
+    set_format_arg(2, uint32_t, peep->id);
```

Comment:
For anyone wondering what this code does it sets the global args to the peep name so that the peep name is correctly written at the top of the window.